src/colmap/controllers/incremental_pipeline.cc

```diff
@@ -326,20 +427,6 @@ IncrementalPipeline::Status IncrementalPipeline::InitializeReconstruction(
   return Status::SUCCESS;
 }
 
-bool IncrementalPipeline::CheckRunGlobalRefinement(
```

Comment:
Same as above, moving it is likely not necessary.

Reply:
Tried to undo. Hope it's more readable now.